tests/test_muonclip_controller.py

```
@@ -0,0 +1,92 @@
+"""Smoke tests for the MuonClipController skeleton."""
+
+import torch.nn as nn
+
+import torch
+import torch.nn as nn
```

Ruff (0.14.4)

Redefinition of unused `nn` from line 3

Remove definition: `nn`

(F811)